frontend/src/components/Gallery/AppBar/Platform/Base.vue

```diff
@@ -60,4 +60,7 @@ const auth = storeAuth();
 #gallery-app-bar {
   z-index: 999 !important;
 }
+.platform-icon {
```

Comment:
This was made since some platform icons are hard to see in the dark toolbar background